frontend/src/components/ShootWorkers/ManageWorkers.vue

```diff
@@ -172,10 +219,13 @@ export default {
     },
     getWorkers () {
       const workers = map(this.internalWorkers, internalWorker => {
-        return omit(internalWorker, ['id', 'valid'])
+        return omit(internalWorker, ['id', 'valid', 'isNewWorker'])
```

Comment:
how about using Symbols?

Reply:
what do you mean? Instead of id?

Reply:
instead of omitting the properties use symbols for `id`, `valid` and `isNewWorker`

Reply:
let's wait for the feedback of @holgerkoser

Reply:
Symbols should be used for private properties in classes to avoid name clashes in subclasses. If we only want the property not to be enumerable, we should use `Reflect.defineProperty`

Reply:
so shall we change this now or leave like it is?